packages/colorpicker/lib/colorpicker.dart

```diff
@@ -0,0 +1,116 @@
+library colorpicker;
```

Comment:
Library file in lib folder contains only library line and exports of files of the package. Check out how other packages are structured. Everything else gets placed in lib/src